### PR TITLE
Add ability to customize working directory of forked process

### PR DESCRIPTION
**Pull request by GrigorievNick**

### CODE REVIEW

**Comment by artimasites** [comment]
@cla-bot[bot] check

**Comment by GrigorievNick** [comment]
Who can review this?

**Comment by bvenners** [comment]
@GrigorievNick Tnx for the contribution. Sorry for the delay in reviewing. I've assigned it to Chee Seng so he'll look at it soon.

File: pom.xml [hunk 1]
```diff
@@ -73,6 +73,19 @@
...
+  <distributionManagement>
+    <repository>
+      <id>central</id>
+      <name>AppsFlyer Artifactory-releases</name>
+      <url>https://artifactory.appsflyer.com:443/artifactory/maven</url>
+    </repository>
+    <snapshotRepository>
+      <id>snapshots</id>
+      <name>AppsFlyer Artifactory-snapshots</name>
+      <url>https://artifactory.appsflyer.com/artifactory/maven-snapshots/</url>
```
Comment (cheeseng) [inline pom.xml:85]: @GrigorievNick Just curious, do we need any artifact from snapshots?

File: pom.xml [hunk 2]
```diff
@@ -215,6 +228,9 @@
           <execution>
             <id>sign-artifacts</id>
             <phase>verify</phase>
+            <configuration>
+              <skip>true</skip>
```
Comment (cheeseng) [inline pom.xml:232]: @GrigorievNick Do you mind to enlighten us why you need to set the skip to true?

File: pom.xml (hunk 1 above)
Comment (GrigorievNick) [reply pom.xml:85]: @cheeseng I am sorry I just deploy this to my own artifactory.
and accidentally commit custom pom details.

File: pom.xml (hunk 2 above)
Comment (GrigorievNick) [reply pom.xml]: @cheeseng I am sorry I just deploy this to my own artifactory.
and accidentally commit custom pom details.

**Review by cheeseng — APPROVED** [review]
@bvenners @GrigorievNick This PR looks good to me, thanks!

**Comment by GrigorievNick** [comment]
Any news on it, I don't want to keep my custom fork in Company's corporate repo for too long.

**Pull request merged**